### ISSUE TITLE
Symlinkined paths like `stable` and `latest` paths dont work

### ISSUE THREAD

**Issue by rafaqz** [LuxDL/DocumenterVitepress.jl#81]
These get a 404:

https://rafaqz.github.io/DimensionalData.jl/stable/
https://rafaqz.github.io/DimensionalData.jl/latest/

But `dev` works because it actually has a folder in the file system
https://rafaqz.github.io/DimensionalData.jl/dev/

I'm not sure what to do about this.

**Comment by asinghvi17** [comment]
Https://asinghvi.github io/SwarmMakie.jl/stable was just built yesterday and seems to work - I have no clue where this is coming from for certain repos!

**Comment by rafaqz** [comment]
Your repos are all switching to the version url, for some reason DD is trying to keep the pretty url `/stable` and it fails.

I guess the only difference is DD has a custom vitepress config?

**Comment by rafaqz** [comment]
Or is there some other setting?

**Comment by rafaqz** [comment]
It happens here too! https://luxdl.github.io/DocumenterVitepress.jl/stable/

**Comment by asinghvi17** [comment]
hmm, it might be the custom config - let me diff the DV and DD ones with the template, which seems to operate correctly...

**Comment by lazarusA** [comment]
<img width="787" alt="Screenshot 2024-03-25 at 20 39 05" src="https://github.com/LuxDL/DocumenterVitepress.jl/assets/19525261/2644fb35-a7f1-42b0-9e24-10c454c22739">
it looks like one of those REPLACE_ME things is still not getting applied properly.

**Comment by lazarusA** [comment]
not the cause, also happens in the good landing page:
<img width="790" alt="Screenshot 2024-03-25 at 20 43 39" src="https://github.com/LuxDL/DocumenterVitepress.jl/assets/19525261/7e1b1ed5-7d8e-4a67-aea0-a3b55aa81d1b">

**Comment by asinghvi17** [comment]
Yeah that's just the description, should be fixed in #80

**Comment by lazarusA** [comment]
<img width="789" alt="Screenshot 2024-03-25 at 20 50 34" src="https://github.com/LuxDL/DocumenterVitepress.jl/assets/19525261/5c5a4b38-d923-4f4f-b31c-8dd494d6ec04">

the ones that work don't have that issue of REPLACE_ME.

**Comment by avik-pal** [comment]
It seems repos that don't have Hydration error do the symlinks correctly. But all the ones which have the error DimensionalData, Lux and Documentervitepress don't work
![image](https://github.com/LuxDL/DocumenterVitepress.jl/assets/30564094/f11cd1b6-5560-4c07-983c-5d35b9bc74fc)
![image](https://github.com/LuxDL/DocumenterVitepress.jl/assets/30564094/85914d09-f423-4f39-a0e2-87eb8c5a7c66)

**Comment by lazarusA** [comment]
Ohhh... see https://github.com/vuejs/vitepress/issues/2293. Also, Also the main difference is the Vue component.  Possible fix https://play.vuejs.org/#__SSR__eNp9UctOwzAQ/BXLl7RSlR7gFEIlqCoB4iXoDXOIkm3rktiWvS6Rqvw7fiQlSIAP0WZ2djzjPdIrpdKDBZrRHKFRdYGwYIKQvOKHULhyaQ3KZg0tkgzd95LRBMFgwiiZR/Y80vP5SCQ3peYKiQG0yv3zRkmNZKS20bIhSTpfyia4SLxCnFrQGR1gb65HTypHsiNdLxAnoQ2NCjaFrR3B+1JaKpO9ObctJu8zDwU3k9CYRhIh2mFakMmUXPaZCdlNEpcpmQ0cf7gQoG/WD/dZVE69blD1p5te+LJzQPczCJpSig3fpnsjhUsTJBktXT5eg35SyKUwjGbDZYwWdS0/7wKG2kJ/iZvZQfnxC743rccYfdZgQB+A0VMPC70FjO3V66PzPGo2srK1Y//TfAEja+s9Rtq1FZWzPeIFt7dhM1xs12bVIggzhPJGh5fxbLcvv9q/on/bPUvPw5x7UNp9AXyX3ws=

**Comment by lazarusA** [comment]
This is better. https://github.com/vuejs/vitepress/issues/1918. I don't have  a lot of time today to test it. But, this might  be it.

**Comment by avik-pal** [comment]
Removing the links in the Features https://github.com/LuxDL/Lux.jl/pull/571/files#diff-36c852523b6a7513a10b8a6bae21f1037eaa27484efc9042f75bbda46d9f529b seems to have fixed the hydration error in most of the lux pages, but when redirecting from stable it still has some hydration errors:
![image](https://github.com/LuxDL/DocumenterVitepress.jl/assets/30564094/38aff22b-fe92-49bc-aa8d-5075ceba88a5)

**Comment by avik-pal** [comment]
(wouldn't recommend this) but if someone desperately needs a workaround till it is properly resolved, manually fixing the symlinks before deploying fixes the deployment https://github.com/LuxDL/Lux.jl/blob/daf571ac359fe602ba9ef773c444298c66bb1378/.github/workflows/DeployDocs.yml#L36-L39

**Comment by lazarusA** [comment]
which links is exactly fixing this?

**Comment by avik-pal** [comment]
> which links is exactly fixing this?

Makes all the symlinks into actual directories and recurses through all the files changes for example, `v0.5.32` to `stable`.

**Comment by oameye** [comment]
I am unsure how to resolve this in [my repo](https://nonlinearoscillations.github.io/HarmonicBalance.jl/stable/). I use `julia-docdeploy` to deploy the documtation. Will I have to change my github action to `deploy-pages`?

**Comment by oameye** [comment]
Is [this commit](https://github.com/rafaqz/DimensionalData.jl/commit/a18a79897403f540e963ff7a9fcbe059babc8515) from @lazarusA also a solution?

**Comment by lazarusA** [comment]
no, it didn't worked 😢 .  But, there is a solution here: https://github.com/rafaqz/Rasters.jl/issues/709, this should be documented here, at some point.

**Comment by oameye** [comment]
That solved it. Thanks! <3

**Comment by jkrumbiegel** [comment]
This is fixed with the new build/deploy process in 0.2, introduced in #246. With that, `stable` is a real folder and not a symlink to another version.
